crates/cargo-nuget/src/main.rs

```diff
@@ -0,0 +1,274 @@
+use cargo_toml::{Manifest, Value};
+use futures::future::{BoxFuture, FutureExt};
+use structopt::StructOpt;
+use thiserror::Error;
+
+use std::ffi::OsString;
+use std::io::Read;
+use std::path::{Path, PathBuf};
+
+fn main() {
+    let Opt::Nuget { subcommand } = Opt::from_args();
+    match subcommand {
+        Subcommand::Install(i) => i.perform().unwrap(),
+    }
+}
+
+/// A utility for interacting with nuget packages
```

Comment:
This comment seems out of place.

Reply:
I believe `StructOpt` uses this to display helpful info the user.